tfw

```diff
@@ -151,6 +165,23 @@ Administrivia:
   admin-users
 
 EOUSAGE
+
+  shopt -s nullglob
+  local external_commands=()
+  for path_prefix in ${PATH//:/ }; do
```

Comment:
Note to self: remember this `${PATH//:/ }` usefulness!

Reply:
It's handy, but definitely brittle if one's `PATH` entries include spaces :frowning_face: .... but I'm feeling lazy I guess?